.eslintrc.js

```diff
@@ -23,14 +23,14 @@ module.exports = {
 		"react/prop-types": "off",
 		"no-mixed-spaces-and-tabs": 0,
 		"react/no-unescaped-entities": 0,
-		"indent": [
-			"error",
-			"tab"
-		],
-		"linebreak-style": [
-			"error",
-			"unix"
-		],
+		// "indent": [
+		// 	"error",
+		// 	"tab"
+		// ],
+		// "linebreak-style": [
+		// 	"error",
+		// 	"unix"
+		// ],
```

Comment:
Why did you try to remove all these eslint rules?

You should try to solve those generated errors instead of removing those eslint rules

You should ask for help if you are facing problems that you can solve.

Its just like you are deleting the tests upon observing that your code fails those tests

Reply:
Sorry, I only intended to remove those rules to test locally but I forgot to restore them before readying the PR. I did uncomment them locally but after the last commit. Sorry for the mistake!